src/Controller/Component/RedirectComponent.php

```diff
@@ -70,9 +70,9 @@ public function redirectToAuthenticationPage($url = null)
         } else {
             $route['?'][$this->query_string_redirect] = Component::getController()->request->getRequestTarget();
         }
-        
-        $resp = $this->controller->redirect(Router::url($route));
-        
+
+        $resp = $this->controller->redirect($route);
```

Comment:
You're right 👍. Could you also remove the now-unneeded `use Cake\Routing\Router;` on line 8?

Reply:
Fixed.